lib/js/google-map-embed.js

```diff
@@ -0,0 +1,76 @@
+jQuery( document ).ready( function($) {
```

Comment:
Google Maps is making some API changes in June - https://cloud.google.com/maps-platform/user-guide/

This doesn't affect the pull request, but something to read if you aren't familiar yet. :)